locals.tf

```diff
@@ -46,7 +48,7 @@ locals {
           Sid    = "Allow principals in the account to decrypt log files"
           Effect = "Allow"
           Principal = {
-            AWS = ["*"]
+            AWS = ["arn:${local.partition}:iam::${local.account_id}:root"] ## changed from '*'
```

Comment:
No need for the comment here and bellow.

Reply:
@hugoalmeida264  Addressed